src/encoder/external_encoder.ml

```diff
@@ -143,45 +150,45 @@ let encoder id ext =
         let ret = read () in
         if ret > 0 then
           [{ Duppy.Task.
-               priority = priority ;
-               events   = events ;
-               handler  = pull }]
+             priority = priority ;
+             events   = events ;
+             handler  = pull }]
         else
-         begin
-           h.log#f 4 "Reading task reached end of data";
-           stop (); []
-         end
+          begin
+            h.log#f 4 "Reading task reached end of data";
+            stop (); []
+          end
       with e -> 
         h.log#f 3
           "Error while reading data from encoding process: %s"
           (Printexc.to_string e) ;
         stop (); 
         (if h.params.restart_on_crash then
-          reset_process start_process h
+            reset_process start_process h
          else
-          raise e);
+            raise e);
```

Comment:
The changeset would have been easier to read without those whitespace changes mixed with actual stuff. Plus, the new indentation on this line does not make any sense!